Header header                # 包含时间戳
float32[] distances_gt      # 距离数组
float32[] theta_gt         # 角度数组
float32[6] pose            # [x,y,z,roll,pitch,yaw]